src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -669,7 +669,6 @@ public OpenIdConnectHandler(HttpClient backchannel)
             var responseMessage = await Backchannel.SendAsync(requestMessage);
             responseMessage.EnsureSuccessStatusCode();
             var userInfoResponse = await responseMessage.Content.ReadAsStringAsync();
-            var userInfoEndpointJwt = new JwtSecurityToken(userInfoResponse);
             var user = JObject.Parse(userInfoResponse);
```

Comment:
If the response is signed, then it will be a jwt. content-type will be application/jwt. http://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse